service/standalone/ui_server.go

```diff
@@ -19,30 +19,35 @@ package standalone
 import (
 	"context"
 	"embed"
+	"errors"
 	"fmt"
 	"io/fs"
 	"net/http"
 	"os"
 	"path/filepath"
+
+	"github.com/clyso/chorus/pkg/dom"
 )
 
 //go:embed all:static
 var staticAssets embed.FS
 
-func serveUI(ctx context.Context, port int) error {
+var errDisabled = errors.New("disabled")
```

Comment:
Is `errDisabled` still used or a leftover?

Reply:
done